stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputView.kt

```diff
@@ -206,6 +206,7 @@ public class MessageInputView : ConstraintLayout {
         binding.dismissInputMode.setOnClickListener { dismissInputMode(inputMode) }
         setMentionsEnabled(style.mentionsEnabled)
         setCommandsEnabled(style.commandsEnabled)
+        binding.messageInputFieldView.setAttachmentMaxFileMb(style.maxFileSize)
```

Comment:
Why is it configurable? Are these limits extendable by our BE?

Reply:
The client can use its own backend, which can have different limits.